### PR TITLE
chore(registry): upgrade to oras v0.7.0

### PR DESCRIPTION
**Pull request by jdolitsky**
Also, adding --insecure flag to "registry login" now exposed by oras

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -44,11 +44,15 @@
 
 [[constraint]]
   name = "github.com/deislabs/oras"
-  version = "0.6.0"
+  version = "0.7.0"
+
+[[constraint]]
+  name = "github.com/containerd/containerd"
+  version = "1.3.0-beta.2"
```
Comment (hickeyma) [inline Gopkg.toml]: Do you think containerd 1.3.0 will be released before Helm 3?

**Review by hickeyma — COMMENTED** [review]
Looks good. Question about containerd version. Also, can we test for `insecure` mode?

**Review by thomastaylor312 — APPROVED** [review]
Minor non-blocking nit. Otherwise good to go

**Review by hickeyma — APPROVED** [review]
LGTM, thanks

File: Gopkg.toml (hunk 1 above)
Comment (hickeyma) [reply Gopkg.toml]: Sure, being experimental it should be ok.

**Pull request merged**